CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
  * When Intel RDT feature is not available, its initialization is skipped,
    resulting in slightly faster `runc exec` and `runc run`. (#3306)
+ * Use `anon` + `file` as root cgroup usage for cgroupv2 nodes. This
+   aligns cgroupv2 usage more closely with cgroupv1 reporting. We sum
+   manually because cgroupv2 has no root `memory.current` file. (#3933)
```

Comment:
What is "manually" here?

Reply:
will clarify, meant to say we use `memory.stat` since there is no `memory.current`

Reply:
is that better?